src/lae_site/handlers/create_subscription.py

```diff
@@ -166,32 +158,20 @@ def render(self, request):
         d.addErrback(signup_failed, customer, self._mailer)
         return NOT_DONE_YET
 
-
-
 def signed_up(claim, request):
-    # Return a page notifying the user that they've been charged and their
-    # service is being set up.
-    tmpl = env.get_template('payment_verified.html')
-    request.write(
-        tmpl.render({
-            "productfullname": "Simple Secure Storage Service",
-            "productname":"S4",
-            "activationinfo": claim.describe(env),
-        }).encode('utf-8'),
-    )
+    # Return 200 and text to be added to template on client
+    request.write(claim.describe(env).encode('utf-8'))
     request.finish()
 
-
-
 def signup_failed(reason, customer, mailer):
-    headers = {
-        "From": FROM_ADDRESS,
-        "Subject": "Sign-up error",
-    }
-    logger.failure("Signup failed", reason)
-    mailer.mail(
-        'info@leastauthority.com',
-        'support@leastauthority.com',
-        "A sign-up failed for <%s>." % (customer.email,),
-        headers,
-    )
+  headers = {
+      "From": FROM_ADDRESS,
+      "Subject": "Sign-up error",
+  }
+  logger.failure("Signup failed", reason)
+  mailer.mail(
+      'info@leastauthority.com',
+      'support@leastauthority.com',
+      "A sign-up failed for <%s>." % (customer.email,),
+      headers,
+  )
```

Comment:
Four space indentation for all Python code, please. :)  Trailing newline at the end of the file is nice too.